src/OrbitGl/TimeGraph.cpp

```diff
@@ -883,10 +887,19 @@ void TimeGraph::DrawMarginsBetweenChildren(
   primitive_assembler.AddBox(MakeBox(timeline_margin_pos, timeline_margin_size),
                              GlCanvas::kZValueTimeBar, GlCanvas::kBackgroundColor);
 
-  // Margin between the Tracks and the vertical scrollbar.
-  Vec2 right_margin_pos{GetWidth() - GetRightMargin(), GetPos()[1]};
-  Quad box = MakeBox(right_margin_pos, GetSize() - (right_margin_pos - GetPos()));
-  primitive_assembler.AddBox(box, GlCanvas::kZValueMargin, GlCanvas::kBackgroundColor);
+  // Right margin mask for Timeline.
+  float right_margin_width = GetRightMargin();
+  float right_margin_height = GetHeight() - timeline_ui_->GetHeight();
```

Comment:
Shouldn't this be `GetHeight()`? The margin spans across the timeline and is positioned at y==0, so why subtract the timeline height?

Reply:
Looking at the code above - should this be `GetHeight() - horizontal_slider_->GetHeight()`? The margin ends above the slider.

By the way, what's the reasoning behind the different behaviors of left and right margin (left one goes all the way to the bottom, right one stops above the slider)? Wouldn't it be more consistent to have the horizontal slider also stop at the margin?

Reply:
Thanks for the review @reichlfl . The reasoning is that I found it weird to have the horizontal slider not touch the horizontal slider, feel free to modify. I've done enough work on this PR :-)